storage/sql/crud.go

```diff
@@ -388,21 +388,17 @@ func (c *conn) UpdateKeys(updater func(old storage.Keys) (storage.Keys, error))
 		// server. Test this, and consider adding a COUNT() command beforehand.
 		old := storage.Keys{}
 		err := c.flavor.lockForUpdate(tx, "keys", "id", keysRowID)
+		if err != nil {
+			return fmt.Errorf("get keys: %v", err)
+		}
+
+		old, err = getKeys(tx)
```

Comment:
[nit] Let's ditch line 389 and go with `old, err :=` here, now that the code is rearranged.